src/ch.usi.inf.nodeprof/src/ch/usi/inf/nodeprof/handlers/ExpressionEventHandler.java

```diff
@@ -25,6 +26,7 @@
 public abstract class ExpressionEventHandler extends BaseSingleTagEventHandler {
     final String expressionType;
 
+    @TruffleBoundary
```

Comment:
This change should be backed out. The fix in Graal.js will be merged in approx. one hour. Just update the import and this compilation problem disappears.

Reply:
Hi, this is to fix an inline compilation problem when running lodash tests in svm mode:
```
[truffle] opt fail         sourceFromStacktrace <split-63cc4e89>                       |Reason org.graalvm.compiler.core.common.PermanentBailoutException: Too deep inlining, probably caused by recursive inlining.
== Inlined methods ordered by inlining frequency:
java.lang.Class.getSimpleName() [987]
ch.usi.inf.nodeprof.analysis.ProfilerExecutionEventNode.onReturnExceptional(VirtualFrame, Throwable) [1]
com.oracle.truffle.js.nodes.JavaScriptNodeWrapper.execute(VirtualFrame) [1]
```
Is this the same compilation problem you faced, too?

Reply:
Sorry, I confused `getSimpleName` with `getPublicName`, so this is not the same compilation error. Please ignore my comment.